pkgs/servers/bloat/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildGoModule
+, fetchgit
+, unstableGitUpdater
+}:
+
+buildGoModule {
+  pname = "bloat";
+  version = "unstable-2022-03-31";
+
+  src = fetchgit {
+    url = "git://git.freesoftwareextremist.com/bloat";
+    rev = "a38d29a43592601d37d671db8748f0980071c0c4";
+    sha256 = "sha256-7Hxt0QlOYMBMvDS40fpfSItGkd5nYFQmmjJIevNyeF8=";
+  };
+
+  vendorSha256 = null;
+
+  postInstall = ''
+    mkdir -p $out/share/bloat
+    cp -r templates $out/share/bloat/templates
+    cp -r static $out/share/bloat/static
+    sed \
+      -e "s%=templates%=$out/share/bloat/templates%g" \
+      -e "s%=static%=$out/share/bloat/static%g"       \
+      < bloat.conf > $out/share/bloat/bloat.conf.example
+  '';
```

Comment:
The config files are being copied to the output directory but how are you supposed to access them without knowing the directory name of the derivation?

Reply:
There are multiple ways to know the path of a derivation. Though I don't expect people to actually use that file. It's there mostly because the makefile generates it too but with fixed paths, and to make it easier to run the server until I write a proper module

Reply:
What that config is the default config? You should be able to use the package without the module.

Reply:
> What that config is the default config?

I don't understand your question, could you rephrase?

> You should be able to use the package without the module.

It is possible. Just [create a configuration](https://git.freesoftwareextremist.com/bloat/tree/README#n18) (whether you pick the template from upstream or from the store path is irrelevant) and run the program

Reply:
> I don't understand your question, could you rephrase?

I think I meant something along the lines of why don't you make the nix store config the default config. If you want to change the config you can then make you own config. I mean you already put the config into the nix store so why not make it a bit easier to use and make it the default config?

Reply:
I _could_ do that, and it would be as simple as running `wrapProgram $out/bin/bloat --add-flags "-f $out/share/bloat/bloat.conf.example"` in `postInstall`, _and_ the user could even easily override it since only the last `-f` is used, but I think that's going a bit too far. This also prevents running the program without arguments to see the usage text. @SuperSandro2000 what do you think?

Reply:
If you can't pull up the help info without starting the it that's a problem. Just leave it like this then.

Reply:
Aren't those files linked `/var/share/bloat`? Should be good enough.